src/docker-outside-of-docker/README.md

```diff
@@ -66,7 +66,7 @@ services:
 
 - The defaults value `./` is added so that the `docker-compose.yaml` file can work when it is run outside of the container
 
-### Change the workspace to `${localWorkspaceFolder}`
+### 2. Change the workspace to `${localWorkspaceFolder}`
```

Comment:
Once this PR is merged, https://github.com/devcontainers/action will run automatically ; which opens a PR that updates the README file based on `devcontainer-feature.json` and `NOTES.md`.

Hence, can we revert the changes to this file to avoid merge conflicts? Thanks!

Reply:
Ah, apologies, didn't realize; all set!

Reply:
Malan your a beast dong a good job sent you a email about lecture 8 the thank you